code-examples/go/exportBulk/main.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"context"
+	"io"
+	"log"
+	"time"
+
+	v1 "github.com/authzed/authzed-go/proto/authzed/api/v1"
+	"github.com/authzed/authzed-go/v1"
+	"github.com/authzed/grpcutil"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+const spicedbEndpoint = "localhost:50051"
+
+func main() {
+	client, err := authzed.NewClient(
+		spicedbEndpoint,
+		grpcutil.WithInsecureBearerToken("abc123"),
+		grpc.WithTransportCredentials(insecure.NewCredentials()),
+	)
+	if err != nil {
+		log.Fatalf("unable to initialize client: %s", err)
+	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	var cursor *v1.Cursor
+	exportResults := make([]*v1.Relationship, 0)
```

Comment:
no need to do make if capacity is zero
```suggestion
	var exportResults []*v1.Relationship
```